addons/website_event/static/tests/tours/website_event_align_test.js

```diff
@@ -0,0 +1,58 @@
+import { registry } from "@web/core/registry";
+
+registry.category("web_tour.tours").add("website_event_submenu_align_breadcrumb_tour", {
+    test: true,
+    url: "/event",
+    steps: () => [{
+        content: 'Go on "Submenu Align and Breadcrumb" page',
+        trigger: 'a[href*="/event"]:contains("Submenu Align and Breadcrumb"):first',
+        run: "click",
+    }, {
+        trigger: 'body:has(.o_wevent_event)',
+        content: 'Wait for the page to load',
+    },  {
+        content: 'Open editor',
+        trigger: '.o_frontend_to_backend_edit_btn',
+        run: 'click'
+    }, {
+        trigger: '.o_menu_systray_item.o_edit_website_container > a',
```

Comment:
`.o_menu_systray_item.o_edit_website_container > a` is too vague 
can we write a proper selector for this